lib/request_handlers/metatags.js

```diff
@@ -33,11 +32,15 @@ module.exports = async function (req, res, next) {
       throw new Error('User not found.')
     }
 
-    return Street.findOne({ creator_id: user._id, namespaced_id: namespacedId })
+    return Street.findOne({
+      where: { creator_id: user._id, namespaced_id: namespacedId }
```

Comment:
Let's do this without using the `_id` field. I realize it was used previously, but this shouldn't have happened.